test/command/pss.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { existsSync, readFileSync, unlinkSync, writeFileSync } from 'fs'
+import { Receive } from '../../src/command/pss/receive'
+import { sleep } from '../../src/utils'
+import { invokeTestCli } from '../utility'
+import { getStampOption } from '../utility/stamp'
+
+describe('Test PSS command', () => {
+  const consoleMessages: string[] = []
+  const getLastMessage = () => consoleMessages[consoleMessages.length - 1]
+
+  beforeAll(() => {
+    global.console.log = jest.fn(message => {
+      consoleMessages.push(message)
+    })
+    global.console.error = jest.fn(message => {
+      consoleMessages.push(message)
+    })
+  })
+
+  beforeEach(() => {
+    consoleMessages.length = 0
+  })
+
+  it('should receive sent pss message', async () => {
+    const invocation = invokeTestCli([
+      'pss',
+      'receive',
+      '--bee-api-url',
+      'http://localhost:11633',
+      '--timeout',
+      '10000',
+    ])
+    await sleep(1000)
+    await invokeTestCli(['pss', 'send', '--target', '00', '--data', 'Bzzz Bzzzz Bzzzz', ...getStampOption()])
```

Comment:
These PSS tests will work only in small network, because it is not clear which node will get the `00` prefixed message.
Anyhow, the `swarm-cli` tests only assure the usage of the exposed functionalities can happen as it was inteded.
So it is good as it is just a reminder if we may would change for a bigger test network where not every node sees any other ones.

Reply:
I cannot test now, but from the Bee code it seems to be possible to use empty string as a target too. Not sure how well the swarm-cli would handle that case :)